recipes/mykrobe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: mykrobe
+  version: "0.5.6"
+
+source:
+  fn: mykrobe-0.5.6.tar.gz
+  url: https://pypi.python.org/packages/7e/35/a662ec560cfc91f35519e1459de917001ca19a9dbefd53c5b3a2b14122d0/mykrobe-0.5.6.tar.gz
+  md5: 5e912497f98cc192e3f0fbf426210560
+
+build:
+  number: 0
+  entry_points:
+      - mykrobe = mykrobe.mykrobe_predictor:main
+      
+requirements:
+  build:
+    - python
+    - setuptools
+
+
+  run:
+    - python
+    - mykatlas
+    - ga4ghmongo
+    - pyvcf
+    - mongoengine
+    - biopython
+    - future
+test:
+  # Python imports
+  imports:
+    - mykrobe
+  commands:
+    - mykrobe --help
+
+
+about:
+  home: https://github.com/iqbal-lab/Mykrobe-predictor
+  license: Custom non-commercial license
+  license_file: https://github.com/iqbal-lab/Mykrobe-predictor/blob/master/LICENSE
```

Comment:
> Since using pipy, the license_file is not part of the source url

Bummer! I opened up an issue for this at the project's repository: https://github.com/iqbal-lab/Mykrobe-predictor/issues/142.
Could you then use GitHub as the source for now, i.e., use:
```yaml
source:
  fn: mykrobe-0.5.6.tar.gz
  url: https://github.com/iqbal-lab/Mykrobe-predictor/archive/933301d9f1e673a2a282255426a885ef72e67c07.tar.gz
  sha256: adcf75a8e873192047921947ef6c367fadee9dff09637f0acad7a4c9c4bcf53e
```
and
```yaml
  license_file: LICENSE
```

Reply:
Changed to github source